apps/web/src/data/subgraph/queries/dashboardQuery.graphql

```diff
@@ -0,0 +1,27 @@
+query dashboard($where: DAOTokenOwner_filter, $first: Int, $skip: Int) {
```

Comment:
Unrelated to this PR, but in testing was able to create a DAO that will not let me start next auction -- https://testnet-nouns-builder-dtm5o0tlr-nouns-builder.vercel.app/dao/goerli/0xEf26f8fb33Df8CE1d0bD9Db70F5ec4D0C1DBa7BD/0

it also doesn't appear in my dashboard -- may be worth opening a new ticket for this

Reply:
Definitely!

Any idea about the steps you took to get here? Haven't seen this one before.